schema.graphql

```diff
@@ -1264,6 +1279,7 @@ type Mutation {
   createConnection(input: CreateConnectionInput!): CreateConnectionResult!
   updateConnection(input: UpdateConnectionInput!): UpdateConnectionResult!
   deleteConnection(input: DeleteConnectionInput!): DeleteConnectionResult!
+  generateDatabaseNewPassword(input: GenerateDatabaseNewPasswordInput!): GenerateDatabaseNewPasswordResult!
```

Comment:
WDYT of `generateNewDatabasePassword` ?

Reply:
Ha, it's already merged in the backend.. Too bad.

Reply:
I was hesitating between the two but that's not a problem I can open another PR for that generateNewDatabasePassword (should be fast)